docs/resources/cbr_policy.md

```diff
@@ -0,0 +1,107 @@
+---
+subcategory: "Cloud Backup and Recovery (CBR)"
+---
+
+## Example Usage
+
+### backup policy
+
+```hcl
+variable "policy_name" {}
+
+resource "huaweicloud_cbr_policy" "test" {
+  name                    = var.policy_name
+  operation_type          = "backup"
+  retention_duration_days = 20
+
+  trigger_pattern {
+    frequency    = "WEEKLY"
+    days         = "MO,TH"
+    backup_times = ["06:00"]
+  }
+}
+```
+
+### replication policy
+
+```hcl
+variable "policy_name" {}
+variable "project_name" {}
```

Comment:
the variable is misleading.

Reply:
thanks. now, it use "dest_region_name" and "dest_project_id".